docs/source/guide/lre-5-theory.md

```diff
@@ -0,0 +1,85 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+
+```{warning}
+The user guide for LRE in Mitiq is currently under construction.
+```
+
+# What is the theory behind LRE?
+
+Layerwise Richardson Extrapolation (LRE), an error mitigation technique, introduced in
+{cite}`Russo_2024_LRE` works by creating multiple noise-scaled variations of the input
+circuit such that the noiseless expectation value is extrapolated from the execution of each
+noisy circuit.
```

Comment:
I think this description is too easily confusable with ZNE. Can we say something like the following?

```md
LRE is an EM strategy which extends ideas found in ZNE by allowing users to create many noise-scaled circuits by folding individual layers instead of the entire circuit.
```

It should use some more tweaking.